src/model_config_utils.h

```diff
@@ -86,6 +86,14 @@ Status GetNormalizedModelConfig(
     const std::string& model_name, const std::string& path,
     const double min_compute_capability, inference::ModelConfig* config);
 
+/// Auto-complete the instance count based on instance kind and backend name.
+/// For KIND_CPU the default instance count is 2 except for the pytorch backend.
```

Comment:
currently this says (except for the pytorch backend).

maybe either 
1. add full list of unsupported backends (pytorch, openvino, ...) 
2. or the converse (default is 2 for TRT/TF/ORT)
3. or just say something more generic (default instance count is 2 for certain backends that benefit from it) so this documentation makes sense even as we change the backends that default to 2 over time

Reply:
>For KIND_CPU the default instance count is 2 except for the pytorch backend.

Would you be ok with me taking this sentence out entirely? After the changes which have been made, I don't think this explanation makes much sense on the API

Reply:
I think it's OK to remove here if we document something about this in server docs: https://github.com/triton-inference-server/server/pull/4466#discussion_r897251607

Reply:
Since the first sentence states that it auto-complete's on the kind and the backend, I'll update the docs (linked) to be more explicit about this feature.